designer/server/src/test/scala/pl/touk/nussknacker/ui/api/MigrationApiHttpServiceSecuritySpec.scala

```diff
@@ -83,7 +82,7 @@ class MigrationApiHttpSecuritySpec
           .jsonBody(requestData)
           .post(s"$nuDesignerHttpAddress/api/migrate")
           .Then()
-          .statusCode(401)
+          .statusCode(500)
```

Comment:
wtf? :)

Reply:
Shame on me, I should mark somehow for further investigation. Short story short: after merging master today this started returning `500`. I had a lot to do and since I like green color in CI I decided to change it for a moment. I should add some commentary for that.  Sorry 4 that